mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule Oidcc.Mixfile do
     [
       app: :oidcc,
       version: to_string(@props[:vsn]),
-      elixir: "~> 1.15",
+      elixir: ">= 1.14.4 and < 2.0.0",
```

Comment:
Why not this?

```suggestion
      elixir: “~> 1.14",
```

Reply:
That would allow `1.14.3`, which does not support OTP 26.

Reply:
@paulswartz Ah right. But you could theoretically also use 1.14.4 with OTP 25, no?

Reply:
@paulswartz Forget the last comment. That makes sense.

Reply:
You could, but the Erlang side of the code wouldn't work. I don't think there's a way to specify supported OTP versions in `mix.exs` unfortunately.